ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-release-1.0.yaml

```diff
@@ -127,6 +138,50 @@ tests:
     - chain: kueue-operator-post
     test:
     - chain: kueue-operator-test-e2e
+- as: test-e2e-disconnected
+  minimum_interval: 96h
+  steps:
+    cluster_profile: gcp
+    env:
+      CATSRC_IMG: registry.redhat.io/redhat/redhat-operator-index:v4.19
+      CATSRC_NAME: redhat-operators
+      MIRROR_OPERATORS: "true"
+    post:
+    - as: kueue-must-gather-disconnected
+      cli: latest
+      commands: |
+        source "${SHARED_DIR}/env"
```

Comment:
I can't use the existing chain here because we're using a different image, which isn't available in the disconnected environment. Here, I'll be using the one from the bundle itself. That's why, instead of a make command, I'll use the individual command to run it.